awx_collection/tests/integration/targets/workflow_job_template/tasks/main.yml

```diff
@@ -461,23 +458,27 @@
 
     - name: Fail if absent and no identifier set
       workflow_job_template_node:
+        identifier: approval_test
```

Comment:
the task mentions "no identifier" but then it is set here, so not sure if that is intended